engine/apps/mobile_app/migrations/0002_alter_mobileappauthtoken_user.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.18 on 2023-03-08 08:45
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('user_management', '0008_organization_is_grafana_incident_enabled'),
+        ('mobile_app', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
Minor suggestion: consider a `RunPython` operation that deletes duplicates before running `AlterField`

Reply:
Good idea 👍 addressed in [11376e5](https://github.com/grafana/oncall/pull/1482/commits/11376e57e3afeeb2e166ea9d42a8b2cfd21b2dcf).

**Before**
![Screenshot 2023-03-08 at 13 40 02](https://user-images.githubusercontent.com/9406895/223716131-3aa9a6f1-02b9-4a89-be39-f7fb8640191f.png)

**After**
![Screenshot 2023-03-08 at 13 40 55](https://user-images.githubusercontent.com/9406895/223716143-8b772bd2-3fd3-444c-8fdc-ed3e0d446a84.png)